fastf1/core.py

```diff
@@ -1005,7 +1015,7 @@ def t0_date(self):
         return self._get_property_warn_not_loaded('_t0_date')
 
     def load(self, *, laps=True, telemetry=True, weather=True,
-             livedata=None):
+             livedata=None, messages=True):
```

Comment:
Please update the docstring. This new argument needs to be explained as well. Also, please keep the livedata kwarg last, so that the arguments are grouped together logically.

Reply:
Changes made and committed